diff --git a/demo/index.html b/demo/index.html
index a2faf28..716ade8 100644
--- a/demo/index.html
+++ b/demo/index.html
@@ -36,6 +36,7 @@
   </style>
 </head>
 <body>
+<!--
 <script type="application/javascript">
   ConsentCookie.init({
     'general': {
@@ -95,5 +96,103 @@
   });
 
 </script>
+-->
+<script>window.ConsentCookie.init({
+  'general': {
+    'uuid': 'd1e3f440-93d3-11e8-9873-fbe73290b77d',
+    'generatedBy': 'dev-demo_1.2.2',
+    'consent': {
+      'type': 'purpose'
+    }
+  },
+  'apps': {
+    'consent': {
+      // ConsentCookie
+      'cc': {
+        'initstate': 'optin',
+        'dataProcessings': [],
+      },
+      // Facebook
+      'fbp': {
+        'initstate': 'optout',
+        'dataProcessings': [],
+      },
+      // Google Tag Manager
+      'gtm': {
+        'initstate': 'alwayson',
+        'dataProcessings': [
+          'gtm-ga'
+        ],
+      },
+      // Hotjar
+      'htj': {
+        'initstate': 'optout',
+        'dataProcessings': [
+          'htj-heatmap',
+          'htj-reqruit',
+          'htj-feedback',
+          'htj-survey',
+          'htj-polls',
+          'htj-form',
+          'htj-conv',
+          'htj-rec'
+        ],
+      },
+      // Iqnomy
+      'iq': {
+        'initstate': 'optout',
+        'dataProcessings': [],
+      },
+      // Zopim chat
+      'zop': {
+        'initstate': 'alwayson',
+        'dataProcessings': [
+          'chat',
+          'contactform'
+        ],
+      },
+      // Google Analytics
+      'ga': {
+        'initstate': 'alwayson',
+        'dataProcessings': [
+          'ga'
+        ],
+      },
+      // Google Adsense
+      'gas': {
+        'initstate': 'optout',
+        'dataProcessings': [],
+      },
+      // Bing Ads
+      'bia': {
+        'initstate': 'alwayson',
+        'dataProcessings': [
+          'bia-user',
+          'bia-conversion',
+          'bia-device',
+          'bia-location',
+          'bia-remarketing',
+          'bia-uet'
+        ],
+      },
+      // MailChimp
+      'mc': {
+        'initstate': 'optout',
+        'dataProcessings': [],
+      },
+      // TradeTracker
+      'tt': {
+        'initstate': 'optin',
+        'dataProcessings': [],
+      },
+      // Wiqhit
+      'wiq': {
+        'initstate': 'alwayson',
+        'dataProcessings': [],
+      }
+    }
+  }
+});
+</script>
 </body>
 </html>
diff --git a/src/components/applications/ccApplicationGroup.vue b/src/components/applications/ccApplicationGroup.vue
new file mode 100644
index 0000000..4c419b3
--- /dev/null
+++ b/src/components/applications/ccApplicationGroup.vue
@@ -0,0 +1,65 @@
+<!--
+  - Copyright 2018 Asknow Solutions B.V.
+  -
+  - Licensed under the Apache License, Version 2.0 (the "License");
+  - you may not use this file except in compliance with the License.
+  - You may obtain a copy of the License at
+  -
+  -     http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing, software
+  - distributed under the License is distributed on an "AS IS" BASIS,
+  - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  - See the License for the specific language governing permissions and
+  - limitations under the License.
+  -
+  -->
+
+<template>
+  <div class="cc-application-group">
+    <cc-application-group-summary :title="groupName"></cc-application-group-summary>
+  </div>
+</template>
+
+<script>
+
+  import utils from 'base/utils';
+
+  // Components
+  import ccApplicationGroupSummary from 'components/applications/ccApplicationGroupSummary.vue';
+
+  // Public functions
+  export default {
+    name: 'cc-application',
+    components: {
+      ccApplicationGroupSummary,
+    },
+    props: {
+      group: {
+        type: Object,
+        required: true,
+      },
+    },
+    data() {
+      return {};
+    },
+    computed: {
+      groupName() {
+        return utils.getObjectValue(this.group, 'definition.name', '');
+      }
+
+    }
+  };
+</script>
+
+<style lang="scss" scoped>
+
+  @import '../../assets/scss/general-variables';
+
+  .cc-application-group {
+
+    @include default-content-border();
+    margin: 10px;
+  }
+
+</style>
diff --git a/src/components/applications/ccApplicationGroupSummary.vue b/src/components/applications/ccApplicationGroupSummary.vue
new file mode 100644
index 0000000..6d38f00
--- /dev/null
+++ b/src/components/applications/ccApplicationGroupSummary.vue
@@ -0,0 +1,112 @@
+<!--
+  - Copyright 2018 Asknow Solutions B.V.
+  -
+  - Licensed under the Apache License, Version 2.0 (the "License");
+  - you may not use this file except in compliance with the License.
+  - You may obtain a copy of the License at
+  -
+  -     http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing, software
+  - distributed under the License is distributed on an "AS IS" BASIS,
+  - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  - See the License for the specific language governing permissions and
+  - limitations under the License.
+  -
+  -->
+
+<template>
+  <div class="cc-application-group-summary">
+    <cc-toggle v-model="showGroup" class="cc-toggle-text">
+      <span>{{ title }}</span>
+    </cc-toggle>
+    <cc-switch v-model="accepted"  :on-title="$t('general.on')" :off-title="$t('general.off')"/>
+  </div>
+</template>
+
+<script>
+  // Components
+  import ccImg from 'components/general/ccImg.vue';
+  import ccToggle from 'components/general/ccToggle.vue';
+  import ccToggleIcon from 'components/general/ccToggleIcon.vue';
+  import ccSwitch from 'components/general/ccSwitch.vue';
+
+  // Public functions
+  export default {
+    name: 'cc-application',
+    components: {
+      ccImg,
+      ccToggle,
+      ccToggleIcon,
+      ccSwitch,
+    },
+    props: {
+      title: {
+        type: String,
+        required: true,
+      },
+    },
+    data() {
+      return {
+        showGroup:false,
+      };
+    },
+    computed:{
+      accepted: {
+        get() {
+          return false;
+        },
+        set($newVal) {
+          console.log($newVal);
+        },
+      },
+    }
+  };
+</script>
+
+<style lang="scss" scoped>
+
+  @import '../../assets/scss/general-variables';
+
+  .cc-application-group-summary {
+
+    display: flex;
+    align-items: center;
+    position: relative;
+    text-align: left;
+    height: 60px;
+
+    @include default-clearfix();
+
+    .cc-toggle-text {
+      display: flex;
+      align-items: center;
+      flex: 1;
+      font-size: 13px;
+      font-weight: 600;
+      margin-left: 10px;
+
+      span {
+        margin-left: 5px;
+      }
+    }
+
+    .cc-toggle-icon {
+      cursor: pointer;
+
+      &.cc-off {
+        color: $cc-brand-color;
+      }
+
+      &.cc-disabled {
+        display: none;
+      }
+    }
+
+    .cc-switch {
+      margin: 15px 10px;
+      display: inline-block;
+      float: right;
+    }
+  }
+</style>
diff --git a/src/components/general/ccSwitch.vue b/src/components/general/ccSwitch.vue
index 9b41e1f..6efc47f 100644
--- a/src/components/general/ccSwitch.vue
+++ b/src/components/general/ccSwitch.vue
@@ -20,6 +20,7 @@
     <span v-theme="{background:'primary'}">
       <div class="cc-state cc-on">{{onTitle}}</div>
       <div class="cc-state cc-off" v-theme="{color:'primary'}">{{offTitle}}</div>
+      <div class="cc-state cc-off" v-theme="{color:'primary'}">{{offTitle}}</div>
       <div class="cc-switch-toggle" v-theme="{borderColor:'primary'}"/>
     </span>
   </label>
@@ -41,7 +42,7 @@
   export default {
     name: 'cc-switch',
     props: {
-      value: Boolean,
+      value: [Boolean,Object],
       disabled: Boolean,
       onTitle: {
         type: String,
diff --git a/src/services/applicationsService.js b/src/services/applicationsService.js
index 3619e0d..b713752 100644
--- a/src/services/applicationsService.js
+++ b/src/services/applicationsService.js
@@ -35,6 +35,10 @@ class ApplicationGroup {
     this.definition = $groupDefinition;
     this.items = _.isArray($items) ? $items : [];
   }
+
+  add($item) {
+    this.items.push($item);
+  }
 }
 
 class ApplicationList {
diff --git a/src/views/applications.vue b/src/views/applications.vue
index abf16fd..227bc18 100644
--- a/src/views/applications.vue
+++ b/src/views/applications.vue
@@ -65,12 +65,18 @@
         get () {
           return this.$services.applications.getApplicationListAsync();
         },
+        // The computed proporty `blogPostContent` will have
+        // the value 'Loading...' until the first time the promise
+        // returned from the `get` function resolves.
         default: null,
       },
     },
     beforeMount() {
       this.$store.commit('updateView', { title: constants.CONFIG_KEY_RESOURCES_APPLICATIONS_TITLE });
     },
+    mounted() {
+      window.steven = this;
+    }
   };
 </script>
 
@@ -102,3 +108,4 @@
     }
   }
 </style>
+0
